services/worker/src/worker/utils.py

```diff
@@ -35,3 +38,169 @@ class PreviousJob(TypedDict):
     dataset: str
     config: Optional[str]
     split: Optional[str]
+
```

Comment:
Good. At one point, all this code could go to libcommon, so that we can also use it in service/api for /rows. I think @lhoestq was working on it, right?

Reply:
Yea I'll open a PR for image+audio support in /rows, and this requires the api to have `datasets`, `Pillow` and these utilities to be in libcommon ^^